extensions/exegol/src/utils/cli.tsx

```diff
@@ -0,0 +1,125 @@
+import { exec } from "child_process";
+import { promisify } from "util";
+import { ExegolContainer } from "../models/ExegolContainer";
+import { readFile, writeFile } from "fs/promises";
+import { join } from "path";
+import { homedir } from "os";
+import { getPreferenceValues } from "@vicinae/api";
+import { existsSync } from "fs";
+import { DEFAULT_CONFIG } from "./config";
+
+const execPromise = promisify(exec);
+
+const EXEGOL_PROFILES_PATH = "/.exegol/profiles.json";
```

Comment:
make sure this exists, or handle it gracefully. Right now it just shows as empty 
<img width="778" height="492" alt="image" src="https://github.com/user-attachments/assets/ae644bf4-e09a-4d71-9fef-9c19abb665b3" />